packages/utils/regl-renderer/src/geometry-utils-V2/path2ToGeometries.js

```diff
@@ -0,0 +1,42 @@
+const path2ToGeometries = (solid, options) => {
+  const defaults = {
+    color: [1, 0.4, 0, 1] // default color
+  }
+  let {color} = Object.assign({}, defaults, options)
+
+  if ('color' in solid) color = solid.color
+
+  let points = solid.points
+
+  let positions = []
+  let prevpoint = undefined
+  points.forEach((point) => {
+    if (prevpoint) {
+      positions.push([prevpoint[0], prevpoint[1], 0])
+      positions.push([point[0], point[1], 0])
+    }
+    prevpoint = point
+  })
+  // add the last segment if necessary
+  if (solid.isClosed && prevpoint) {
```

Comment:
It may be possible, but I don't really want to rework all the code. Can you add this to the list? Should open a new issue for someone to do some research?

Reply:
sure !